diagnostics-extension/tsconfig.json

```diff
@@ -3,11 +3,9 @@
   "compilerOptions": {
     "target": "es6",
     "lib": [
-      "ES2015",
+      "ES6",
```

Comment:
Let's stick with the official name here.

https://bytearcher.com/articles/es6-vs-es2015-name/